pkg/sql/delegate/show_table.go

```diff
@@ -171,6 +171,21 @@ func (d *delegator) delegateShowConstraints(n *tree.ShowConstraints) (tree.State
 	return d.showTableDetails(n.Table, getConstraintsQuery)
 }
 
+func (d *delegator) delegateShowCreateAllTables() (tree.Statement, error) {
+	sqltelemetry.IncrementShowCounter(sqltelemetry.Create)
+
+	const showCreateAllTablesQuery = `
+	SELECT crdb_internal.show_create_all_tables(%[1]s);
```

Comment:
nit: should we make it `SELECT ... AS statements` or something so that the result has a column name that we like?

Reply:
Any suggestions on what the column should be called?
cc @vy-ton

Reply:
Just to conform for `crdb_internal.show_create_all_tables` the output is a single column table with one row? And we're looking for the column name?

Reply:
Yep, more specifically a column name for the `SHOW CREATE ALL TABLES` command which is just an alias for `crdb_internal.show_create_all_tables`

Reply:
`database_tables`

Reply:
for context, the name of the corresponding column of `SHOW CREATE TABLE x` is `create_statement`

Reply:
Thanks, I should have checked that as well. Reusing `create_statement` makes sense to me, the `SHOW` command already describes the output